src/main/java/org/example/service/JsonResponseService.java

```diff
@@ -0,0 +1,48 @@
+package org.example.service;
+
+import java.io.PrintWriter;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+public class JsonResponseService {
+
+    private final ConnectionService connectionService = ConnectionService.getInstance();
+
+    private static final String NAME_OF_TABLE = "developer";
+
+    private static JsonResponseService INSTANCE = null;
```

Comment:
Remove null.

Check the Java conventions for naming:

- variables
- methods
- classes
- constant variables
- class files

Reply:
fix